### PR TITLE
Add rluvaton to collaborator section

### PR DESCRIPTION
**Pull request by rluvaton**
As requested in #969 by @goldbergyoni 

> I think I'll squash and merge to remove all the trial and error commits

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1358,9 +1358,9 @@ Thank you to all our collaborators! 🙏
...
 
-| <a href="https://github.com/idori" target="_blank"><img src="assets/images/members/ido.png" width="75" height="75"></a> | <a href="https://github.com/TheHollidayInn" target="_blank"><img src="assets/images/members/keith.png" width="75" height="75"></a> |
-| :---------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------: |
-|                                    [Ido Richter (Founder)](https://github.com/idori)                                    |                                        [Keith Holliday](https://github.com/TheHollidayInn)                                         |
+| <a href="https://github.com/idori" target="_blank"><img src="assets/images/members/ido.png" width="75" height="75"></a> | <a href="https://github.com/TheHollidayInn" target="_blank"><img src="assets/images/members/keith.png" width="75" height="75"></a> | <a href="https://github.com/rluvaton" target="_blank"><img src="assets/images/members/raz-luvaton.png" width="75" height="75"></a> |
```
Comment (elektronik2k5) [inline README.md:1361]:
```suggestion
| <a href="https://github.com/idori" target="_blank"><img src="assets/images/members/ido.png" width="75" height="75"></a> | <a href="https://github.com/TheHollidayInn" target="_blank"><img src="assets/images/members/keith.png" width="75" height="75"></a> | <a href="https://github.com/rluvaton" target="_blank"><img src="assets/images/members/raz-luvaton.png" width="75" height="75" alt="Raz Luvaton" loading="lazy"></a> |
```

File: README.md [hunk 2]
```diff
@@ -1358,9 +1358,9 @@ Thank you to all our collaborators! 🙏
...
+| :---------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------: | :--------------------------------------------------------------------------------------------------------------------------------: |
+|                                    [Ido Richter (Founder)](https://github.com/idori)                                    |                                        [Keith Holliday](https://github.com/TheHollidayInn)                                         |                                        [Raz Luvaton (Full Stack Developer)](https://github.com/rluvaton)                                         |
 
 ### Collaborator Emeriti
```
Comment (elektronik2k5) [inline README.md:1366]: This comment is about the next file: `assets/images/members/raz-luvaton.png` (cause GitHub doesn't allow commenting on binary files 🤦🏻‍♂️ ).
1. Considering the fact the image is rendered at 75x75, I'd replace that asset with a smaller one.
2. JPG is more suitable for photos and will compress better than PNG.

**Comment by goldbergyoni** [comment]
@rluvaton Something breaks in the UI, see img

**Comment by rluvaton** [comment]
> @rluvaton Something breaks in the UI, see img
> 
> 
> 
> image
> 
> 

It's because the title, I can remove it or break it to a new line

File: README.md (hunk 2 above)
Comment (rluvaton) [reply README.md:1366]: > 1. Considering the fact the image is rendered at 75x75, I'd replace that asset with a smaller one.

I've tried many websites and sizes but the image looked blurry so I kept the image size

> 2. JPG is more suitable for photos and will compress better than PNG.

Done!

**Comment by rluvaton** [comment]
For now, I removed the title, but just for comparison this is how would it look if it had the title but with a new line
new-line

Current:
without-title

File: README.md (hunk 2 above)
Comment (elektronik2k5) [reply README.md:1366]: The 100k JPEG is 7 times smaller than the original PNG.

**Review by elektronik2k5 — APPROVED** [review]

**Pull request merged**